docker/couchdb/monitor.sh

```diff
@@ -1,7 +1,9 @@
 #!/bin/bash
 
 # start couchdb exporter
-nohup couchdb-prometheus-exporter -telemetry.address=":9984" 2>&1 1>& couchdb_exporter.log < /dev/null &
+COUCH_CONFIG=/data/srv/current/auth/couchdb/couchdb_config.ini
+nohup couchdb-prometheus-exporter -telemetry.address=":9984" -logtostderr=true \
```

Comment:
Well, in my view you should print appropriate message but there is no need to start exporter which will fail every cycle it will query couchdb. Doing this way we'll have appropriate message in a log and no exporter.